Makefile

```diff
@@ -81,7 +81,8 @@ e2e-test: deploy
 # Test Runners
 # ------------
 test:
-	CGO_ENABLED=1 go test -race -v ./...
+	CGO_ENABLED=1 go test -race -v $$(go list ./... | grep -v test)
```

Comment:
```suggestion
	CGO_ENABLED=1 go test -race -v ./pkg/...
```

perhaps?

Reply:
Yeah, I think it is more clear and better :)